app/controllers/api/functions.php

```diff
@@ -899,7 +899,7 @@
             }
 
             if(!$current->isEmpty()) {
-                $jwtObj = new JWT(App::getEnv('_APP_OPENSSL_KEY_V1'), 'HS256', 900, 10); // Instantiate with key, algo, maxAge and leeway.
+                $jwtObj = new JWT($project->getId() == 'console' ? App::getEnv('_APP_OPENSSL_KEY_V1') : $project->getAttribute('jwtSecret'), 'HS256', 900, 10); // Instantiate with key, algo, maxAge and leeway.
```

Comment:
Should we add JWT as a resource we initialize in one place?

Reply:
as mentioned above, we can also try to cast the console attributes in a higher level to avoid code repetition.